ci/jobs/projects.yaml

```diff
@@ -92,6 +92,8 @@
         - 2.10-nightly
         - 2.11-beta
         - 2.11-nightly
+	- 2.12-beta
```

Comment:
I think this is marked as red because a Tab character is used instead of spaces. I think your editor is probably set to using tabs instead of spaces. Could you please change this to spaces so it's consistent with the rest of the file? Thanks!